.gitignore

```diff
@@ -1,12 +1,18 @@
 bazel-*
 .bazelrc.user
-node_modules
+node_modules/
+.pnpm-*
 
-# Don't commit lockfile for now as it is unstable. Do allow for it to be
-# created, however, since it gives a performance boost for local development.
-# https://github.com/bazelbuild/bazel/issues/19026
-# https://github.com/bazelbuild/bazel/issues/19621
-# https://github.com/bazelbuild/bazel/issues/19971
-# https://github.com/bazelbuild/bazel/issues/20272
-# https://github.com/bazelbuild/bazel/issues/20369
+.idea/
+.ijwb/
+.vscode
+.DS_Store
+
+# Bazel's MODULE lockfile isn't ready to check in yet as of Bazel 7.1.
+# Do allow for it to be created, however, since it gives a performance boost for local development.
+# [Store resolved repository attributes in the Bzlmod lockfile](https://github.com/bazelbuild/bazel/issues/19026)
```

Comment:
Do you have something in an IDE that actually renders this?

Reply:
Nope. Tho still an improvement over the raw PR#s IMO